### DISCUSSION TITLE
About branch

### DISCUSSION THREAD

**Discussion by prachi0011** [Lakhankumawat/LearnCPP#1126] [Q&A] [answered]
**While adding files to repository, we have to create new branch or upload in main branch?**

**Comment by Lakhankumawat** [comment, marked as answer]
hi @prachi0011 you can create a new branch from main on your fork and make the changes in it for a new pr ,
so whenever you need to make a pr create a new branch from your fork's main.

Also do not forget to fetch upstream on main before pushing some code or creating a new branch ,
